CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## New Features
 
 - Added support for `RelationalGroupedDataframe.pivot()` to access `pivot` in the following pattern `Dataframe.group_by(...).pivot(...)`.
+- Added experimental feature local testing.
```

Comment:
Added experimental feature: Local Testing Mode.

Reply:
I think we need more for this part, can copy paste some from our doc.
I added one line just to pass the test